modules/using-wildcard-routes.adoc

```diff
@@ -0,0 +1,286 @@
+// Module included in the following assemblies:
+//
+// * networking/configuring-ingress-controller
+
+[id="using-wildcard-routes_{context}"]
+= Using wildcard routes
+
+The HAProxy Ingress Controller has support for wildcard routes. The Ingress Operator uses `wildcardPolicy` to configure the `ROUTER_ALLOW_WILDCARD_ROUTES` environment variable.
+
+The default behavior of the Ingress Controller is to admit routes with a wildcard policy of `None`, which is backwards compatible with existing `IngressController` resources.
+
+.Procedure
+
+. Configure the wildcard policy.
+.. Use the following command to edit the `IngressController` resource.
++
+----
+$ oc edit IngressController
+----
++
+.. Under `spec`, set the `wildcardPolicy` field to `WildcardsDisallowed` or `WildcardsAllowed`:
++
+[source,yaml]
+----
+spec:
+  routeAdmission:
+    wildcardPolicy: WildcardsDisallowed # or WildcardsAllowed
+----
+
+
+.Samples for using a secure wildcard edge terminated route
```

Comment:
This is all content that was brought forward from 3.11. Does it still have value? If not, I can go ahead and remove it. 

In the meantime, the references to router (mostly - not in the code blocks) have been changed to Ingress Controller, and the old `oc adm` commands have been removed.

Reply:
I chose to comment out the samples section until I'm certain that it is valid for 4.x.